fastlane/metadata/es-ES/release_notes.txt

```diff
@@ -0,0 +1,25 @@
+Hemos resuelto un problema por el que los sitios autoalojados se bloqueaban al intentar cargar medios sin conexión a Internet. Un gran problema, pero una gran corrección.
+
+Hablando de medios, sabemos que podrías haberle dado a la aplicación un acceso limitado a tu biblioteca de medios. No es un gran problema, pero hemos añadido un aviso al selector de medios en caso de que quieras cambiar tus ajustes y permitir más acceso. El aviso viene incluso en tipo dinámico para facilitar la lectura.
+
+También hemos hecho que VoiceOver y Dynamic Type sean más precisos en los calendarios de registro de actividades y de entradas programadas. Nos complace informar que esto mejora la experiencia del usuario para las personas que necesitan una mejor accesibilidad visual.
+
+Grandes noticias para los comentarios: Ahora puedes seguir una conversación usando un botón en la cabecera de los comentarios y moderar los comentarios individuales a través de un menú desplegable.
```

Comment:
Notice the translation still include the copy about the comment threads moderation. It's okay at this point because the translation fixes probably haven't been addressed yet. 

I'll make sure to keep an eye out for them when finalizing the release ~~later this week~~ tomorrow (gosh! is it the end of the week already?!)